src/api/app/controllers/webui/image_templates/interconnects_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Webui
+  module ImageTemplates
+    class InterconnectsController < Webui::WebuiController
+      def index
+        @projects = Project.image_templates
+        respond_to do |format|
+          format.xml
```

Comment:
If this is just rendering xml then why is the controller in the Webui namespace?